app/invocation/invocation_execution_card.tsx

```diff
@@ -260,6 +261,15 @@ export default class ExecutionCardComponent extends React.Component {
         incompleteCount++;
       }
     }
+    console.log(this.props.filter);
+    const filteredActions = this.state.executions.filter(
+      (action) =>
+        !this.props.filter ||
+        `${action.actionDigest.hash}/${action.actionDigest.sizeBytes}`
```

Comment:
Seems useful to be able to filter by the command, e.g. `go`, `bash`, `tsc`, ... if it doesn't slow down the search by too much, mind adding that here too?

Reply:
I added that and made the placeholder for the filter input a bit different so it's clear what the user can filter by